composer.json

```diff
@@ -20,7 +20,8 @@
     },
     "autoload": {
         "psr-0": {
-            "TS\\Writer\\": "src/"
+            "TS\\Writer\\": "src/",
```

Comment:
This namespace also needs to be renamed then?

Reply:
For source, we will do it of course but not here. We may do that on least priority